installer-guide/mac-install.md

```diff
@@ -5,73 +5,40 @@
+![](../images/installer-guide/mac-install-md/munki.png)
 
-![](../images/installer-guide/mac-install-md/gib-process.png)
+As you can see, we get a nice list of macOS installers. If you need a particular versions of macOS, you can select it by typing the number next to it. For this example we'll choose 10:
+
+![](../images/installer-guide/mac-install-md/munki-process.png)
 
 * **macOS 11, Big Sur Note**: As this OS is quite new, there's still some issues with certain systems to resolve. For more information, see here: [OpenCore and macOS 11: Big Sur](../extras/big-sur/README.md)
   * For first time users, we recommend macOS 10.15, Catalina
 * **Nvidia GPU Note**: Reminder to verify whether your hardware support newer OSes, see [Hardware Limitations](../macos-limits.md)
 
 This is going to take a while as we're downloading the entire 8GB+ macOS installer, so it's highly recommended to read the rest of the guide while you wait.
 
-Once finished, we have to either extract the installer or build it:
-
-* [Extracting the installer](#extracting-the-installer)
-  * For macOS 11+
-* [Building the installer](#building-the-installer)
-  * For 10.15 and older
-
-### Extracting the installer
-
-For macOS 11 and newer, Apple now packages up the installer into the InstallAssistant package. This will be located under `gibMacOS/macOS Downloads/`:
-
-![](../images/extras/big-sur/readme/final-download.png)
-
-Run the InstallAssistant.pkg and point this to whichever drive you're booting off of, this is where the Install.app will be dropped:
+Once finished, you'll find in your `Users` folder a DMG containing the macOS Installer, called like `Install_macOS_11.1-20C69.dmg`. Mount it and you'll find the installer application.
 
-![](../images/extras/big-sur/readme/install-pkg.png)
+![](../images/installer-guide/mac-install-md/munki-done.png)
 
-Once done, you should find it located in your Applications folder:
-
-![](../images/extras/big-sur/readme/done.png)
+![](../images/installer-guide/mac-install-md/munki-dmg.png)
 
 From here, jump to [Setting up the installer](#setting-up-the-installer) to finish your work.
-
-### Building the installer
-
-For macOS 10.15 and older, the installer will be downloaded in pieces and will need to be built. Here we'll want to run the `BuildmacOSInstallApp.command`:
-
-![](../images/installer-guide/mac-install-md/gib-location.png)
-
-You will be prompted for the macOS installer files which were downloaded to `macOS Downloads` folder in the gibMacOS directory.
-
-From the Finder, drill down to the folder containing the downloaded files and either drag it to the command line or "Cmd+C" and paste it to the terminal.
-
-Once the task is completed, exit the utility. You will find the Install file in the directory.
-
-Move the newly created image to the Applications folder – this will simplify the next section.
```

Comment:
I recommend still telling users to move it to the `/Applications` folder as users will try to execute `createinstallmedia` without correcting the path

Other solution would be to edit the below `createinstallmedia` commands, however would get complicated for people with installers pulled directly from the App Store or Apple's archival site